smart-contracts/contracts/merkle-incentives/src/msg.rs

```diff
@@ -7,7 +7,9 @@ use crate::{
 };
 
 #[cw_serde]
-pub struct InstantiateMsg {}
+pub struct InstantiateMsg {
+    pub incentive_admin: String,
```

Comment:
Annoying comment, this should be `incentives_admin`, i think we always refer to it in the plural form.

Reply:
sounds good, will change